src/linter_test.cc

```diff
@@ -16,40 +16,100 @@
 
 #include "src/linter.h"
 
-#include "gtest/gtest.h"
+#include <string>
+#include <fstream>
+#include <sstream>
+#include <cstdio>
 
+#include "gtest/gtest.h"
 
-namespace zetasql {
+namespace zetasql::linter {
 
-namespace linter {
+namespace {
 
 TEST(LinterTest, StatementLineLengthCheck) {
     const absl::string_view sql =
         "SELECT e, sum(f) FROM emp where b = a or c < d group by x";
 
-    EXPECT_TRUE(checkLineLength(sql).ok());
-    EXPECT_FALSE(checkLineLength(sql, 10).ok());
-    EXPECT_TRUE(checkLineLength(sql, 10, ' ').ok());
+    EXPECT_TRUE(CheckLineLength(sql).ok());
+    EXPECT_FALSE(CheckLineLength(sql, 10).ok());
+    EXPECT_TRUE(CheckLineLength(sql, 10, ' ').ok());
 }
 
-
 TEST(LinterTest, StatementValidityCheck) {
     const absl::string_view valid_sql = "SELECT 5+2";
 
     const absl::string_view invalid_sql = "SELECT 5+2 sss ddd";
 
-    EXPECT_TRUE(checkStatement(valid_sql).ok());
-    EXPECT_FALSE(checkStatement(invalid_sql).ok());
+    EXPECT_TRUE(CheckStatement(valid_sql).ok());
+    EXPECT_FALSE(CheckStatement(invalid_sql).ok());
 
-    EXPECT_TRUE(checkStatement(
+    EXPECT_TRUE(CheckStatement(
         "SELECT * FROM emp where b = a or c < d group by x").ok());
 
-    EXPECT_TRUE(checkStatement(
+    EXPECT_TRUE(CheckStatement(
         "SELECT e, sum(f) FROM emp where b = a or c < d group by x").ok());
 
-    EXPECT_FALSE(checkStatement(
+    EXPECT_FALSE(CheckStatement(
         "SELET A FROM B\nSELECT C FROM D").ok());
 }
 
-}  // namespace linter
-}  // namespace zetasql
+TEST(LinterTest, SemicolonCheck) {
+    const absl::string_view valid_sql =
```

Comment:
I'd just inline those test cases. Then test failure message will contain the problematic sql.

Reply:
Inlined that, but I think I eventually need to seperate them, since functions getting more complex.